moj-node/test/utils/adapters.spec.js

```diff
@@ -194,7 +194,7 @@ function videoContent() {
     },
     secondaryTags: [],
     categories: [],
-    establishmentId: 792,
+    establishmentIds: [792],
```

Comment:
Would it be worth checking a piece of content with multiple IDs? As that triggered the discovery of this bug, it might help prevent a regression.